picard/cluster.py

```diff
@@ -213,14 +217,14 @@ def lookup_metadata(self):
         if self.lookup_task:
             return
         self.tagger.window.set_statusbar_message(
-            N_("Looking up the metadata for cluster %(album)s..."),
-            {'album': self.metadata['album']}
+                N_("Looking up the metadata for cluster %(album)s..."),
+                {'album': self.metadata['album']}
         )
         self.lookup_task = self.tagger.mb_api.find_releases(self._lookup_finished,
-            artist=self.metadata['albumartist'],
-            release=self.metadata['album'],
-            tracks=string_(len(self.files)),
-            limit=QUERY_LIMIT)
+                                                            artist=self.metadata['albumartist'],
+                                                            release=self.metadata['album'],
+                                                            tracks=string_(len(self.files)),
+                                                            limit=QUERY_LIMIT)
```

Comment:
Nope, instead of inserting tons of spaces, wrap first arg and indent with 4 spaces.
I hate when everything is pushed on the right for no reason.